sponge_intellij_style.xml

```diff
@@ -0,0 +1,134 @@
+<code_scheme name="Sponge">
```

Comment:
Probably worth putting this into `.gitignore`

Reply:
I can do but isn't this the code format you want contributions to follow? I feel there would be value in including it in an easily accessible place. I had to redownload it myself after a fresh OS install.

Reply:
Fair point. Let's leave it there then.